.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2.1
+
+jobs:
+  Deploy to GitHub pages:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "f1:3e:05:3a:66:69:60:4f:1f:9f:8f:af:6e:1f:0f:53"
```

Comment:
@vladikoff did you help set this up on the app-services repo?

Reply:
@eoger I think?

Reply:
This was @linacambridge actually

Reply:
Yep, that was me! I'll weigh in on this after my 1:1 with @rfk. 😆